pkgs/development/python-modules/pytest-workflow/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, jsonschema
+, pytest
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-workflow";
+  version = "2.0.1";
+  pyproject = true;
+
```

Comment:
The package should also indicate that it requires at least Python 3.7 (e.g. through `pythonOlder`).